index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
- [ ] Please use the new version of Microverse readme, AKA: Microverse [Readme](https://github.com/microverseinc/readme-template) template should be used.